### PR TITLE
Add a Twitter account setting for sharing

### PR DESCRIPTION
**Pull request by MikkCZ**
Fix https://github.com/mozilla/One-Mozilla-blog/issues/16

### CODE REVIEW

**Comment by MikkCZ** [comment]
@craigcook you are right, it's already there. Here is the simplest patch without changing any single line of JS. If you prefer to simplify how it looks at the same time, just tell me your idea. I would be thinking of to make it less JavaScripty, personally ;)

**Comment by MikkCZ** [comment]
Oh, I have committed wrong code, wrong input used.

File: themes/OneMozilla/functions.php [hunk 1]
```diff
@@ -201,6 +213,19 @@ function onemozilla_settings_field_share_pages() { ?>
 }
 
 /**
+ * Renders the Twitter account setting field to share via.
+ */
+function onemozilla_settings_field_tweet_at() { ?>
+	<div class="layout tweet-at">
+	<label>
+		<input type="checkbox" id="onemozilla_tweet_at" name="onemozilla_tweet_at" value="<?php echo get_option('onemozilla_tweet_at', 'firefox'); ?>" />
```
Comment (craigcook) [inline themes/OneMozilla/functions.php]: this should be input `type="text"`

Comment (MikkCZ) [inline themes/OneMozilla/functions.php]: Sorry, my bad. Should be fixed now.

**Comment by MikkCZ** [comment]
@craigcook r?

**Comment by craigcook** [comment]
Argh! So sorry I neglected this for so long. I'm a terrible person 😞 

I've squashed your two commits and merged them in e23ce3da32ef260572b5906bf7b82ee565cd6313

**Comment by MikkCZ** [comment]
No problem. Thank you for merging this. ;)

**Pull request closed without merging**